consensus-types/types/payload_header.go

```diff
@@ -59,7 +66,7 @@ type ExecutionPayloadHeader struct {
 	// ExtraData is the extra data of the block.
 	ExtraData bytes.Bytes `json:"extraData"`
 	// BaseFeePerGas is the base fee per gas.
-	BaseFeePerGas *uint256.Int `json:"baseFeePerGas"`
+	BaseFeePerGas *math.U256 `json:"baseFeePerGas"`
```

Comment:
I see `math.U256` is an alias for `uint256.Int` so formatting should not have changed here